src/store/index.ts

```diff
@@ -971,6 +976,9 @@ export default new Vuex.Store({
     setReturnToReview({ commit }, shouldReturn: boolean) {
       commit("doSetReturnToReview", shouldReturn);
     },
+    setArrivedFromStep5({ commit }, shouldReturn: boolean) {
+      commit("doSetArrivedFromStep5", shouldReturn);
```

Comment:
maybe change var `shouldReturn` here and in the mutation to something like `didArriveFromStep5`